README.md

```diff
@@ -2,314 +2,77 @@
 
 # Cloudinary Client Side JavaScript Library
 
-Cloudinary is a cloud service that offers a solution to a web application's entire image management pipeline.
-
-Easily upload images to the cloud. Automatically perform smart image resizing, cropping and conversion without installing any complex software. Integrate Facebook or Twitter profile image extraction in a snap, in any dimension and style to match your website’s graphics requirements. Images are seamlessly delivered through a fast CDN, and much much more.
-
-Cloudinary offers comprehensive APIs and administration capabilities and is easy to integrate with any web application, existing or new.
-
+Cloudinary is a cloud service that offers a solution to a web application's entire image management pipeline.</br></br>
+Easily upload images to the cloud. Automatically perform smart image resizing, cropping and conversion without installing any complex software. Integrate Facebook or Twitter profile image extraction in a snap, in any dimension and style to match your website’s graphics requirements. Images are seamlessly delivered through a fast CDN, and much much more.</br></br>
```

Comment:
you don't really need the BR tags in markdown. you can use double spaces (2 spaces at end of line)